### PR TITLE
Implement 'find' feature to query for matching clients

### LINKED ISSUE #103: As a property manager, I can search for client based on certain details so that I can narrow down my search

### PR DESCRIPTION
**Pull request by wilsonngja**
Fix #103

### CODE REVIEW

**Review by OVReader — COMMENTED** [review]
Overall LGTM

File: src/main/java/seedu/duke/Messages.java [hunk 1]
```diff
@@ -166,4 +166,18 @@ public class Messages {
             + "\ne/ for client email"
             + "\nb/ for client budget";
     public static final String MESSAGE_BYE = "Goodbye :). See you soon!";
+
+
+    /* Error message for Find Properties */
+
+    public static final String MESSAGE_INVALID_FIND_SUBCOMMANMD = "OOPS!!! You are either missing a subcommand or your"
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/Messages.java]: Please use the standardised subcommand message

File: src/main/java/seedu/duke/Messages.java [hunk 2]
```diff
@@ -166,4 +166,18 @@ public class Messages {
...
+            + " subcommand might in the wrong location. Please re-type in the correct format as shown below:"
+            + "\n    find -client f/Doja Cat";
+
+    public static final String MESSAGE_FIND_NO_DESCRIPTION = "OOPS!!! Seems like you forgot to include a tag."
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/Messages.java]: Please use the standardised no description message

File: src/main/java/seedu/duke/parsermanager/ParseFindClient.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            String currentString = commandSplitByTag[i];
+            isEmptyString = currentString.equals(EMPTY_TEXT);
+            if (!isEmptyString) {
+                queryLists.add(currentString);
+            }
+        }
+
+        boolean hasCorrectNumOfTag = queryLists.size() == SINGLE_TAG;
+        boolean hasCorrectTag = commandDescription.contains(FIND_FLAGS);
+        boolean isValidTag = hasCorrectTag & hasCorrectNumOfTag;
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/parsermanager/ParseFindClient.java]: & is a bitwise operator

**Review by FeliciaBeatrice — APPROVED** [review]
everything looks good! LGTM

**Pull request merged**